libs/blocks/src/lib/features/content-left/content-left.classnames.ts

```diff
@@ -0,0 +1,31 @@
+import { cva } from 'class-variance-authority';
+
+export const container = cva(
+  'flex flex-col gap-gap-xl lg:gap-gap-lg justify-center',
+  {
+    variants: {
+      content: {
+        'content-left': 'lg:flex-row-reverse',
```

Comment:
can we change it to `left` and  `right` since `content` is already the parent of the variant

Reply:
because the variant type is content-left and content-right if I change it will affect the section that uses this block. 
<img width="555" alt="image" src="https://github.com/deriv-com/deriv-com-v2/assets/142988136/4d28cfb8-48b4-4c6f-b177-bafe1ac40001">

Otherwise, I needed to change to this to handle the type 
`className={container({
          content: variant === 'content-left' ? 'left' : 'right',
        })}`. Right now is `className={container({ content: variant })}`

Reply:
ok then instead of `content` can we change it to `variant` instead?

Reply:
we can keep `content-left` and `content-right` if we do it like that.

Reply:
I changed it to variant instead of using content. Thanks @prince-deriv